content/events/2016-09-13-realitime-apps-with-node.js-and-rethinkdb.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Realtime Apps with Node.js and RethinkDB
+type: event
+date: 2016-09-13
+starttime: 18:00
+endtime: 20:00
+presenter: Matt Collins
+presenter_img: https://avatars3.githubusercontent.com/u/795287?v=3
+presenter_bio: |
```

Comment:
I've often wondered about speakers who have returned to speak - should we re-use the bio? I think I had settled on it being an "of the moment" part of content which I think should be fresh each time. What do you think?

Reply:
I think it makes more sense to be new each time because things might have changed since the last time they spoke, and I wouldn't want to change the bio from a past time which would have been correct at the time.